### ISSUE TITLE
Install problem

### ISSUE THREAD

**Issue by sunscreem** [gioppy/statamic-metatags#1]
Hey - I just installed this, it's showing up in the control panel but I get this when I click on `settings` or `defaults`.

![image](https://user-images.githubusercontent.com/7523595/103314750-ca16db00-4a1b-11eb-831a-def885494b60.png)

Here's the stack trace for this:

https://flareapp.io/share/LPdj16z5

Maybe I did something dumb?

FYI - This is how it appears in the add-ons:

![image](https://user-images.githubusercontent.com/7523595/103314767-d56a0680-4a1b-11eb-8b15-18ae066aae8c.png)

```
Statamic 3.0.36 Solo
Laravel 8.20.1
PHP 7.4.12
gioppy/statamic-metatags 1.0.2
```

Let me know what you need from me to debug it :)

**Comment by gioppy** [comment]
Hi @sunscreem , it is very strange. I have now tried to activate the addon on a clean Statamic installation and I have not found any errors when viewing the Settings or Default page. The error is very strange and makes me think that something went wrong while installing Statamic or addon. Try deleting the vendor folder and re-running the `composer install` command.

**Comment by sunscreem** [comment]
Hey @gioppy  - still the same I'm afraid.

Happy to zip this repo up and send it over if you fancy trying to recreate it?

**Comment by gioppy** [comment]
Ok, send me your zip so i can check what is the problem.

**Comment by sunscreem** [comment]
Hey @gioppy that's it sent. Happy new year :)

**Comment by gioppy** [comment]
Hi @sunscreem, thanks for zip archive. I downloaded the zip and installed all the modules from composer. I see you have a few custom configurations (DB, S3, etc), which don't make the front-end work for me, but it's still not a problem. I installed my addon from the Statamic interface and it works without problems. I see you use Sail for the development environment: maybe there are some problems on Sail with Statamic. I use Valet for my local environment and I have no problems. I tried to set up Sail but I have some problems and it seems I can't get it to work.
